dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -141,16 +141,16 @@ JSON_FIELDS_FORMAT_WARNING.explanation=The JSON logging field name mapping is in
 JSON_FIELDS_FORMAT_WARNING.useraction=Ensure the format of the JSON logging field name mapping property is correct. The JSON field name mapping can be specified in a number of ways, specified in the logging configuration documentation.
 
 JSON_FIELDS_FORMAT_WARNING_2=TRAS3008W: The JSON logging field name mapping property has too many or too few tokens in an entry.
-JSON_FIELDS_FORMAT_WARNING_2.explanation=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or eventType:knownFieldName:newFieldName.
-JSON_FIELDS_FORMAT_WARNING_2.useraction=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or eventType:knownFieldName:newFieldName.
+JSON_FIELDS_FORMAT_WARNING_2.explanation=The JSON logging field name mapping property should be formatted knownFieldName:newFieldName or source:knownFieldName:newFieldName.
```

Comment:
should we change 'knownFieldName' to 'defaultFieldName' (applies to explanation and useraction)?